ambry-router/src/main/java/com.github.ambry.router/PutOperation.java

```diff
@@ -631,7 +671,7 @@ private void prepareForSending() {
     void onFillComplete() {
       buf.flip();
       prepareForSending();
-      chunkReadyTimeMs = time.milliseconds();
+      chunkReadyAtMs = time.milliseconds();
```

Comment:
ChunkState makes this clear. Ready means ready to send. Free or Building means fillable.